src/screens/PostScreen.tsx

```diff
@@ -7,37 +8,67 @@ import { styles } from './styles';
 export default function PostScreen() {
   return (
     <View style={styles.container}>
-      <ProfilePlaceholder />
-      <Text>rbeggs</Text>
-      <Text>September 19</Text>
-      <Text>
+      {/* Profile Header */}
+      <View style={styles.profileHeader}>
+        <ProfilePlaceholder style={styles.profileIcon} />
+        <View>
+          <Text style={styles.profileName}>rbeggs</Text>
+          <Text style={styles.postDate}>September 19</Text>
+        </View>
+      </View>
```

Comment:
Given that we want the username + profile placeholder to be grouped together, let's wrap them in their own div. Then apply `justifyContent: 'space-between'` to the `profileHeader` style to spread information across the width of the screen.